jsuite/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# pull official base image
+FROM python:3.9
```

Comment:
can we add an editor (maybe nano cuz we're not vim overlords) to the container by default pls?
https://stackoverflow.com/questions/30853247/how-do-i-edit-a-file-after-i-shell-to-a-docker-container

Reply:
Why do you want an editor? All the relevant files are mounted with volumes no?

Reply:
because I am blind and did not see them.